src/views/dataaccesscoe/pl-planning.vue

```diff
@@ -0,0 +1,127 @@
+    </p>
+
+    <!-- Tab links -->
+    <div class="dh-data-access-coe_tab">
+      <button :class="active_tab=='id_tab1'?'dh-data-access-coe_tab-active':''" @click="tabClicked($event, 'id_tab1')">Program/Project Manager</button>
+      <button :class="active_tab=='id_tab2'?'dh-data-access-coe_tab-active':''" @click="tabClicked($event, 'id_tab2')">Data Providers</button>
+    </div>
+
+    <!-- Tab content -->
+    <div v-if="active_tab=='id_tab1'" id="id_tab1" class="usa-layout-docs__main desktop:grid-col-10 dh-data-access-coe_tab-content">
+      <p>
+        To ensure the Data Access CoE provides the maximum benefit to a project or procurement, we recommend engaging the Data Access CoE team in at the beginning of the project lifecycle. To begin this engagement during the Planning phase, you can find resources on this page and contact us directly at
+        <a href="mailto:data.itsjpo@dot.gov">data.itsjpo@dot.gov<img class="in-line-external-link-icon" src="/images/icons/external-link_1b1b1b.svg" alt="External from DOT link icon" title="External link"></a>.
+      </p>
+      <p>
+        During Planning, the Data Access CoE can support Program/Project Managers by:
+      </p>
+      <ul>
+        <li>
+          Advising on data access requirements clauses and solicitation language for Notice of Funding Opportunities (NOFOs), Requests for Proposals (RFPs) and internal ITS JPO-sponsored projects. We have standard language and clauses that can be tailored to your needs.
+        </li>
+        <li>
+          Determining applicability for data access requirements for your project, including whether data can be made public and what U.S. DOT-managed systems are be available to store project data.
+        </li>
+        <li>
+          Providing templates to begin data management in the solicitation phase (e.g. preliminary data management plans [DMPs] and instructions).
+        </li>
+        <li>
+          Other resources and support upon request. Contact us at
+          <a href="mailto:data.itsjpo@dot.gov">data.itsjpo@dot.gov<img class="in-line-external-link-icon" src="/images/icons/external-link_1b1b1b.svg" alt="External from DOT link icon" title="External link"></a>.
+        </li>
+      </ul>
+      <i>Resources</i>
+      <ul>
+        <li>
+          <router-link to="/resources/guidelines">ITS DataHub Guidelines</router-link> – these are ITS JPO’s overall guidelines and requirements for data access and management.
+        </li>
+        <li>
+          <router-link to="/resources/data-management/preliminary-dmp">Preliminary DMP</router-link> Instructions – a document to be used in a solicitation or early phases of a project to create a plan for data management. This document is filled out by a solicitation or project applicant.
```

Comment:
Instead of `This document is filled out by a solicitation or project applicant.` should probably say `This document is to be filled out by a solicitation or project applicant.`

Reply:
Approval from tech-writer is needed

Reply:
Approved and changes made